### PR TITLE
admin-ajax.phpの影響の調整

### PR DESCRIPTION
**Pull request by mahirock**
#64

暫定処置として、padding-leftと同じ幅のmargin-rightを追記して対応しました。
いかがでしょうか？

### CODE REVIEW

**Comment by gouten5010** [comment]
@mahirock admin-ajax.phpの影響なんてないんですが、何を指しているのでしょう？

**Comment by gouten5010** [comment]
@mahirock marginじゃなくてpaddingにします。
あと、記述場所間違ってますね。

**Comment by mahirock** [comment]
@gouten5010

なんか、いろいろすみません……
